indico/htdocs/sass/partials/_actionboxes.scss

```diff
@@ -84,11 +84,19 @@
     background: $background-color;
     color: $color;
 
-    > .section > .icon {
+    > .section {
```

Comment:
I would take the opportunity to stop restricting action-box elements to be directly nested. (`>`)